pyiron_atomistics/atomistics/job/atomistic.py

```diff
@@ -891,3 +892,15 @@ def __dir__(self):
             return hdf5_path.list_nodes()
         else:
             return []
+
+    @property
+    def mean_squared_displacement(self):
+        """
+        The mean-squared displacement computed directly within Lammps.
+
+        WARNINGS:
+            1. This works only if the job is a `Lammps` job.
+            2. Returns an output ONLY when the `msd` flag in set to True in `calc_md` for a Lammps job.
+            3. Returns None if used with `calc_static`, `calc_minimize` and `calc_vcsgc` for a Lammps job.
+        """
+        return self._job["output/generic/mean_squared_displacement"]
```

Comment:
Do we need to reference the HDF path itself? My understanding is that job should be a live object and we can use the real output path `job.output.mean_squared_displacement`. 

As much as possible we need to break free from strings and explicit dependence on storage implementation details.

Reply:
All the `job.output.<something>` functions reference the HDF path.

Reply:
Suppose we change the storage path. If you explicitly reference the path here, we need to update this code *and* `job.output.<something>`. If you directly reference `job.output.<something>` we only need to change it there. Plus strings are a huge pain in the butt to find and update 😛

Reply:
Yes, yes. But this seems to be the case for all outputs, not just `mean_squared_displacement`.

For eg. `job.output.energy_pot `returns `self._job["output/generic/energy_pot"]`

Reply:
Right, my bad. I see now you're extending `GenericOutput` not the job itself.

Basically, IMO, this code is a hot mess where we have our storage path implementation hard-coded everywhere and transitioning over to using `DataContainer` is going to be a huge headache. That's not your fault, but it means that I don't want to make that eventual transition worse by piling on more baggage...

What *does* need fixing in this PR is that the output field is defined in the wrong place. I didn't appreciate this on the first readthrough, but having it live on the `Generic` level and then explicitly warn that it only works for Lammps jobs is a no-go. 
  The simples solution is to go over to `pyiron_atomistics/lammps/base.py` and introduce a new `_LammpsOutput(GenericOutput)` which extends functionality with this new attribute. I would also then wrap the return in a try/except (not sure the exact error when the HDF field doesn't exist) and return `None` when it's not there, maybe raising a warning then that you had to ask for it. Then your docstring doesn't need all these disclaimers. Don't forget to overwrite `self.output = LammpsOutput(job=self)` in `LammpsBase.__init__`.